linera-core/src/worker.rs

```diff
@@ -1078,3 +1002,26 @@ where
         }
     }
 }
+
+#[cfg(with_testing)]
+impl<StorageClient> WorkerState<StorageClient>
+where
+    StorageClient: Storage,
+    ViewError: From<StorageClient::StoreError>,
+{
+    #[tracing::instrument(level = "trace", skip(self))]
```

Comment:
nit: I believe we always put the comment first.

Reply:
Oh I just took these out of the trait.  I'll reformat :)